pkg/ibmvpc/sg_analysis_test.go

```diff
@@ -107,3 +110,101 @@ func TestSGRule(t *testing.T) {
 	require.Equal(t, sgRule.index, 1)
 	require.Equal(t, ruleStr, "index: 1, direction: inbound,  conns: protocol: all, remote: 0.0.0.0/0, local: 10.240.10.0/32\n")
 }
+
+func TestNoLocalField(t *testing.T) {
+	cidr1t, _ := ipblock.FromCidrOrAddress("10.250.10.1")
+	cidr2t, _ := ipblock.FromCidrOrAddress("10.250.10.0/30")
+
+	c1 := connection.TCPorUDPConnection(netp.ProtocolString("UDP"), 5, 87, 10, 3245)
+	c2 := connection.TCPorUDPConnection(netp.ProtocolString("TCP"), 1, 100, 5, 1000)
+
+	rulesTest1 := []*SGRule{
+		{
+			target:      cidr1t,
+			connections: c1,
+			index:       1,
+			local:       ipblock.GetCidrAll(),
+		},
+		{
+			target:      cidr2t,
+			connections: c2,
+			index:       2,
+			local:       ipblock.GetCidrAll(),
+		},
+	}
+
+	target1, _ := ipblock.FromCidrOrAddress("10.250.10.0")
+	target2, _ := ipblock.FromCidrOrAddress("10.250.10.2")
+	target3, _ := ipblock.FromCidrOrAddress("10.250.10.1")
+
+	egressConnectivityMap := make(map[*ipblock.IPBlock]*ConnectivityResult)
+	mapAndAnalyzeSGRules(rulesTest1, false, egressConnectivityMap)
+	require.Equal(t, len(egressConnectivityMap), 1)
```

Comment:
```suggestion
	require.Equal(t, len(egressConnectivityMap), 1) //expecting size 1 because locals are only "0.0.0.0/0" in this example
```
explain `require` in documentation